g11n-ws/modules/md-restful-api-i18n/src/main/java/com/vmware/vip/i18n/api/base/utils/CommonUtility.java

```diff
@@ -17,14 +17,17 @@ private CommonUtility() {}
      * @param categories
      * @return
      */
-    public static boolean checkParams(String[] categories, String... args){
-        List<String> catList = Arrays.asList(ConstantsKeys.ALL_CATEGORY);
+    public static boolean checkParams(List<String> categories, String... args){
```

Comment:
What's the usage of 'args'?

Reply:
Args is used to verify whether the request parameter is empty. It may be used to verify multiple parameters,such as "CommonUtility.checkParams(categories, language, region)"